merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/ResponseSerializers.java

```diff
@@ -493,14 +493,26 @@ public static JsonValue serializeNoSuchActivityTypeException(final MissionModelS
   public static JsonValue serializeInputMismatchException(final InputMismatchException ex) {
     return Json.createObjectBuilder()
                .add("message", "input mismatch exception")
-               .add("cause", ex.getMessage())
+               .add("extensions", serializeCauseAsExtension(ex.getMessage()))
```

Comment:
Are these the only spot we are sending the exception through via a Hausara action? Just curious if there are other responses on the server that can benefit this.

Reply:
Got it.